README.md

```diff
@@ -7,7 +7,6 @@ OMERO Server
 Installs and configures OMERO.server.
 
 **Warning:** This defaults to Python 3.
-Upgrades from OMERO.server 5.5 Python 2.7 to OMERO.server 5.6+ Python 3.6 are handled.
```

Comment:
Maybe the line above as well? Either omitting it or saying, "only supports"

Reply:
Resolved in fdc5b64